apps/wallet/src/ui/app/pages/home/nft-transfer/index.tsx

```diff
@@ -1,124 +1,68 @@
-    );
     const navigate = useNavigate();
-    const onHandleSubmit = useCallback(
-        async (
-            { to }: FormValues,
-            { resetForm }: FormikHelpers<FormValues>
-        ) => {
-            if (!objectId) {
-                return;
-            }
-            setSendError(null);
-            try {
-                const resp = await dispatch(
-                    transferNFT({
-                        recipient: to,
-                        objectId,
-                        gasBudget: DEFAULT_NFT_TRANSFER_GAS_FEE,
-                    })
-                ).unwrap();
-                resetForm();
-                if (resp.txId) {
-                    navigate(
-                        `/receipt?${new URLSearchParams({
-                            txdigest: resp.txId,
-                            from: 'nfts',
-                        }).toString()}`
-                    );
-                }
-            } catch (e) {
-                setSendError((e as SerializedError).message || null);
-            }
-        },
-        [dispatch, navigate, objectId]
-    );
-    const handleOnClearSubmitError = useCallback(() => {
-        setSendError(null);
-    }, []);
-    const { loading } = useObjectsState();
-    return (
-        <div className={cl(st.container, { 'items-center': loading })}>
-            <Loading loading={loading}>
-                {selectedNft && objectId && hasPublicTransfer(selectedNft) ? (
-                    <>
-                        <PageTitle
-                            title="Send NFT"
-                            back={`/nft-details?${new URLSearchParams({
-                                objectId,
-                            }).toString()}`}
-                        />
-                        <div className={st.content}>
-                            <NFTDisplayCard
-                                nftobj={selectedNft}
-                                wideView
-                                size="sm"
-                            />
 
-                            <Formik
-                                initialValues={initialValues}
-                                validateOnMount={true}
-                                validationSchema={validationSchema}
-                                onSubmit={onHandleSubmit}
-                            >
-                                <TransferNFTForm
-                                    submitError={sendError}
-                                    gasBudget={DEFAULT_NFT_TRANSFER_GAS_FEE}
-                                    onClearSubmitError={
-                                        handleOnClearSubmitError
-                                    }
+    return (
+        <Overlay
+            showModal={true}
+            title="Send NFT"
+            closeOverlay={() => navigate('/nfts')}
+        >
+            <div className="flex w-full flex-col h-full">
+                <Loading loading={isLoading}>
+                    {objectId && nftId ? (
```

Comment:
this is a bit confusing `objectId` and `nftId` is the same thing right?

Reply:
Good catch